package.json

```diff
@@ -47,6 +48,8 @@
     "url": "^0.11.0"
   },
   "dependencies": {
+    "date-fns": "^2.17.0",
```

Comment:
These should be peer dependencies.

Reply:
we had the conversation and going ahead and merging